chapter_01/hinglish/introduction.ipynb

```diff
@@ -0,0 +1,1346 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    decision = True
</pre><p>decision -&gt; condition.</p><p> </p><p>you take decision based on the condition.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/abhiabhi94/learn-python/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>good catch.</p>